web3_proxy/src/app/mod.rs

```diff
@@ -128,6 +170,7 @@ impl DatabaseReplica {
 
 /// Cache data from the database about rpc keys
 pub type RpcSecretKeyCache = Arc<CacheWithTTL<RpcSecretKey, AuthorizationChecks>>;
+pub type UserBalanceCache = Arc<CacheWithTTL<u64, Decimal>>; // Could also be an AtomicDecimal
```

Comment:
Other places we use the user id, we use NonZeroU64. I don't know how much that matters though

Reply:
The database saves it as a u64 that why I was using it. I will change it to `NonZeroU64`, let's keep it consistent